Assets/MixedRealityToolkit/Utilities/AssetScriptReferenceRetargeter.cs

```diff
@@ -0,0 +1,593 @@
+                    Directory.Delete(outputDirectory, true);
+                }
+
+                Directory.CreateDirectory(outputDirectory);
+                foreach (Assembly dll in dlls)
+                {
+                    if (dll.name.Contains("MixedReality"))
+                    {
+                        if (!asmDefMappings.TryGetValue($"{dll.name}.dll", out string newDllGuid))
+                        {
+                            throw new InvalidOperationException($"No guid based on .asmdef was generated for dll '{dll.name}'.");
+                        }
+
+                        File.Copy(Path.Combine(tmpDirPath, $"{dll.name}.dll"), Path.Combine(outputDirectory, $"{dll.name}.dll"));
+                        File.Copy(Path.Combine(tmpDirPath, $"{dll.name}.dll.meta"), Path.Combine(outputDirectory, $"{dll.name}.dll.meta"));
+
+                        Object[] assets = AssetDatabase.LoadAllAssetsAtPath(Path.Combine("Assets", temporaryDirectoryName, $"{dll.name}.dll"));
+
+                        foreach (Object asset in assets)
+                        {
+                            MonoScript monoScript = asset as MonoScript;
+                            if (!(monoScript is null) && AssetDatabase.TryGetGUIDAndLocalFileIdentifier(monoScript, out string guid, out long fileId))
+                            {
+                                Type type = monoScript.GetClass();
+
+                                if (type.Namespace == null || !type.Namespace.Contains("Microsoft.MixedReality.Toolkit"))
+                                {
+                                    throw new InvalidDataException($"Type {type.Name} is not a member of the Microsoft.MixedReality.Toolkit namespace");
+                                }
+                                toReturn.Add(type.FullName, new ClassInformation() { Name = type.Name, Namespace = type.Namespace, FileId = fileId, Guid = newDllGuid });
+                            }
+                        }
+                    }
+                }
+
+                return toReturn;
+            }
+            finally
+            {
+                Directory.Delete(tmpDirPath, true);
+                AssetDatabase.Refresh();
+            }
+        }
+
+        private static void PostProcess(string outputPath, Dictionary<string, string> dllGuids)
+        {
+            DirectoryInfo outputDirectory = new DirectoryInfo(outputPath);
+            RecursiveFolderCleanup(outputDirectory);
+            CopyIntermediaryAssemblies(Application.dataPath.Replace("Assets", "NuGet/Plugins"));
+            UpdateMetaFiles(dllGuids);
+        }
+
+        private static void CopyIntermediaryAssemblies(string outputPath)
+        {
+            List<DirectoryInfo> outputDirectories = new List<DirectoryInfo>();
+            string playerDataCachePath = Application.dataPath.Replace("Assets", "Library/PlayerDataCache");
+            DirectoryInfo playerDataCahce = new DirectoryInfo(playerDataCachePath);
```

Comment:
>playerDataCahce [](start = 26, length = 15)

Nit playerDataCache (but also I think that this isn't used, so maybe this line should just be deleted